packages/framework/collabspaces/src/collabSpaces.ts

```diff
@@ -502,9 +531,35 @@ export class TempCollabSpaceRuntime
 	public async getCellDebugInfo(
 		row: number,
 		col: number,
-	): Promise<{ channel?: ICollabChannelCore }> {
+	): Promise<{ channel?: ICollabChannelCore; channelId?: string; rowId: string; colId: string }> {
 		const result = this.getCellInfo(row, col);
-		return { channel: await result.channel };
+		assert(result.channelId !== undefined, "channelId is missing");
+		const { rowId, colId } = this.parseChannelId(result.channelId);
+		const channel = await result.channel;
+		return { channel, channelId: result.channelId, rowId, colId };
+	}
+
+	public getReverseMapCellDebugInfo(rowId: string, colId: string): { row: number; col: number } {
+		const row = this.reverseMap.getRowId(rowId);
+		const col = this.reverseMap.getColId(colId);
+		assert(row !== undefined, "rowIndex is missing");
+		assert(col !== undefined, "colIndex is missing");
+		return {
+			row: row - 1,
+			col: col - 1,
+		};
+	}
+
+	// For test purposes only!
+	// Returns the Reverse Map size and the actual indexes from matrix stored on the reverse mapping matrixes.
+	public getReverseMapsDebugInfo(): {
+		rowMap: { [id: string]: number };
```

Comment:
Consider wrapping return type with Readonly<>, such that caller can't change it.